### PR TITLE
simplify build & release actions

### LINKED ISSUE #23: Images are not pushed to quay.io
the last one is 6 months old https://quay.io/repository/kaotoio/standalone?tab=tags

image

### PR DESCRIPTION
**Pull request by lburgazzoli**
- ci: simplify build & release process

FIxes #23

### CODE REVIEW

File: .github/workflows/build-main.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: Build Main
+
+on:
+  push:
+    branches:
+      - 'main'
+  schedule:
+    - cron:  '12 0 * * *'
```
Comment (lordrip) [inline .github/workflows/build-main.yml:8]: ~[curious-question]: Is it needed to build the image daily?~

Also, does this mean "Daily At 12:12 AM"?

Comment (lordrip) [reply .github/workflows/build-main.yml:8]: Auto answer: Yes, to bring the changes from Kaoto UI and Kaoto Backend that happen on other repositories.

**Review by lordrip — COMMENTED** [review]
I'm not an expert on GitHub actions, but seems about right to me

**Review by igarashitm — APPROVED** [review]

**Comment by igarashitm** [comment]
It looks like the Required PR check workflow is out of sync, anyone has the permission to change required check?

**Pull request merged**